apps/workers/db-check/main.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/jmoiron/sqlx"
+	tools "github.com/pcelvng/task-tools"
+	"github.com/pcelvng/task-tools/bootstrap"
+	"github.com/pcelvng/task-tools/db"
+	"github.com/pcelvng/task/bus"
+)
+
+const (
+	taskType    = "task.dbcheck"
+	description = `
+Runs record count checks against a given table's data to verify that the data is updated as expected
+
+uri params:
+table      - required table name to validate
+date_field - required field name that holds the date value
+date1      - 1st date value to check for records
+date2      - 2nd date to compare row counts between date1 and date2
+offset     - if date2 is not given, this is the offset duration back from date1 to get date2
+percent    - (default 0.05) percent is the allowed percentage of deviation between offset and compare row counts
```

Comment:
Tolerance is a better description of what it does.

Reply:
I like that better I will change this.